opus/application/static_media/js/search.js

```diff
@@ -443,15 +441,16 @@ var o_search = {
     },
 
     adjustSearchWidgetHeight: function() {
-        let containerHeight = $("#search").height() - 80;
+        let footerHeight = $(".app-footer").outerHeight();
+        let mainNavHeight = $("#op-main-nav").outerHeight();
+        let totalNonSearchAreaHeight = footerHeight + mainNavHeight;
+        let containerHeight = $("#search").height() - totalNonSearchAreaHeight;
```

Comment:
do you need to use outerHeight here as well?  <i can't tell; i suspect you are ok but wanted to check>

Reply:
If we use .outerHeight() here, the scrollbar will be able to scroll all the down below the footer (a little bit covered by the footer), so we use .height() here and scrollbar looks fine.